Makefile

```diff
@@ -4,18 +4,16 @@
 
 PREFIX := $(DESTDIR)/usr/local
 BINDIR := $(PREFIX)/sbin
-GIT_BRANCH := $(shell git rev-parse --abbrev-ref HEAD 2>/dev/null)
-GIT_BRANCH_CLEAN := $(shell echo $(GIT_BRANCH) | sed -e "s/[^[:alnum:]]/-/g")
-RUNC_IMAGE := runc_dev$(if $(GIT_BRANCH_CLEAN),:$(GIT_BRANCH_CLEAN))
-RUNC_TEST_IMAGE := runc_test$(if $(GIT_BRANCH_CLEAN),:$(GIT_BRANCH_CLEAN))
+RUNC_IMAGE := runc_dev
+RUNC_TEST_IMAGE := runc_test
 PROJECT := github.com/opencontainers/runc
 TEST_DOCKERFILE := script/test_Dockerfile
 BUILDTAGS := seccomp
 RUNC_BUILD_PATH := /go/src/github.com/opencontainers/runc/runc
 RUNC_INSTANCE := runc_dev
+PKG_NAME := github.com/opencontainers/runc
 COMMIT := $(shell git rev-parse HEAD 2> /dev/null || true)
-RUNC_LINK := $(CURDIR)/Godeps/_workspace/src/github.com/opencontainers/runc
-export GOPATH := $(CURDIR)/Godeps/_workspace
+TEMP_GOPATH := /tmp/runc-gopath
```

Comment:
nit: i dont like this TEMP_GOPATH, but i guess it has values when user doesnt check out to the standard gopath.